62.CAD/main.cpp

```diff
@@ -893,6 +1952,23 @@ struct DrawBuffersFiller
 		currentIndexCount += objectCount * 6u;
 	}
 
+	void addHatchIndices_Internal(uint32_t startObject, uint32_t objectCount)
```

Comment:
u don't need to do this specialized for hatches. remove this function and use `addPolylineObjectIndices_Internal`
and actually rename that to `addCagedObjectIndices_Internal`

FYI our new algorithm allow the index buffer to be pre-filled and not do this for every object. 
previously we had to do this because objects may have had back and front faces for transparency, this is no longer the case. (I will change that in a future milestone)